src/main/java/com/dicemc/corruptedlands/Config.java

```diff
@@ -19,6 +19,8 @@
 	public static ForgeConfigSpec.ConfigValue<Integer> CALYX_RESISTANCE_INFECTED;
 	public static ForgeConfigSpec.ConfigValue<Boolean> CALYX_HARMED_BY_PURIFIER;
 	public static ForgeConfigSpec.ConfigValue<Float> CORRUPTION_EFFECT_POWER;
+	public static ForgeConfigSpec.ConfigValue<String> BIOMERESIST;
```

Comment:
This might be better as an Config list `ForgeConfigSpec.ConfigValue<List<? extends String>>` so that it can be entered like `["minecraft:desert","minecraft:extreme_hills"] and you don't have to parse it.  Since you do have some parsing code though you could use this format plus your parser to add in biome-specific resistances.  so the config would look like `["minecraft:desert;100","minecraft:extreme_hills;50"]`.  Then you just parse the number following the ";" character as your biome-specific resistance value.

Reply:
I'll give it a try and see what happens. May not be able to finish it until later though given I've only started this at 2 am and I won't be working for all that long.

Reply:
Getting a list to present itself in config seems a bit painful, what I may try to do is a little more parsing with that example in mind, similar concept but the string would be typed out as "minecraft:desert;100,minecraft:extreme_hills;50"

First I split the biomes-resistance pairs, then I suppose a map would probably be the best way to store it? I'll work on it

Reply:
The new method does require parsing, however this is all handled at load time in common space, and should bear minimal impact to the game itself once it successfully loads up.